packages/client/hmi-client/src/page/data-explorer/components/tera-search-results-list.vue

```diff
@@ -168,7 +168,7 @@ const projectOptions = computed(() => [
 						// then, link and store in the project assets
 						if (datasetId) {
 							response = await useProjects().addAsset(AssetType.Dataset, datasetId, project.id);
-							assetName = selectedAsset.value.name;
+							assetName = selectedAsset.value.name ?? '';
```

Comment:
I wonder if instead of doing this we can force TerariumAsset to provide an empty string if `name` or `description` is null?

Reply:
Agreed, but from slack conversation, this is part of a much larger cleaning on these data objects that needs to be done in a separate issue: https://github.com/DARPA-ASKEM/terarium/issues/3466